File: src/dashboard/Elsa.WorkflowDesigner/Models/ActivityDefinition.cs
namespace Elsa.WorkflowDesigner.Models
{
    public class ActivityDefinition
    {
        public ActivityDefinition()
        {
            Properties = new ActivityPropertyDescriptor[0];
        }

        public string Type { get; set; }
        public string DisplayName { get; set; }
        public string Description { get; set; }
        public string Category { get; set; }
        public string Icon { get; set; }
        public ActivityPropertyDescriptor[] Properties { get; set; }
        public ActivityDesignerSettings Designer { get; set; }
    }
}

File: src/designer/elsa-workflows-studio/src/modelsCopy/activity-definition.ts
import { ActivityPropertyDescriptor } from "./"

export type Lambda<T = any> = string | T

export interface ActivityDefinitionCopy {
  type: string
  displayName: string
  description?: string
  runtimeDescription?: Lambda<string>,
  category?: string
  icon?: string
  properties?: Array<ActivityPropertyDescriptor>
  outcomes?: Lambda<Array<string>>
}
